src/app/app.module.ts

```diff
@@ -36,7 +46,7 @@ import { LoginComponent } from './login/login.component';
     MatSnackBarModule,
     MatProgressBarModule,
   ],
-  providers: [],
+  providers: [SubsFirestoreService],
```

Comment:
Why is this service injected in the root?

Reply:
I read about injecting to root and it seemed unnecessary in this case. It has been removed.